tailwind.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         dark: "#687182",
         disabled: "#F7F9FC",
         secondary: "#4A4E51",
+        shipment: "#F4F7FC", 
```

Comment:
It cannot have name like this. We will use it several time (for Editable Table - btw here I made mistake and have different color). Maybe `smallTableHeader` or `editableTableHeader` just to be sure for what this color is